legacy/ForceService.js

```diff
@@ -422,6 +422,37 @@ ${JSON.stringify(data)}
     }
     return flatResults
   }
+
+  /** calls APEX REST API to get a Map of supported <CDN Base Url, Label> */
+  async getCDNBaseUrlLabelMap() {
+    return await this.getPimConstantsCDNData('cdnLabelMap');
+  }
+
+  /** calls APEX REST API to get a Map of supported <Label, CDN Base Url> */
+  async getLabelCDNBaseUrlMap() {
+    return await this.getPimConstantsCDNData('labelCDNMap');
+  }
+
+  getPimConstantsCDNData(type) {
+    const accessToken = this.sessionId;
+    return new Promise(function (resolve, reject) {
+      let request = require("request");
+      request({
+          url: 'https://pim-kim-2-dev-ed.develop.my.salesforce.com/services/apexrest/pim/constants?type=' + type,
```

Comment:
not sure we can rely on your dev org to retrieve data. would every org have this endpoint as it's part of the package now. Just use the host name from the connection.